.github/workflows/check-update-pelorus-operator.yml

```diff
@@ -0,0 +1,31 @@
+name: Check for updates for pelorus operator
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 0 * * 1' # Every Monday at 00:00 UTC
+
+jobs:
+  project-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+          cache: "pip"
+          cache-dependency-path: |
+            **/requirements*.txt
+            pyproject.toml
+
+      - name: Check if Pelorus operator was bumped
+        run: |
+          make bump-pelorus-operator
+          git restore pelorus-operator/helm-charts/pelorus/Chart.lock
+          test -z "$(git status --short)"
```

Comment:
I will check how to check if the job failed, and if so, inform a message like "Updates can be made to pelorus operator. Run `make bump-pelorus-operator` to update it."